webdriver-extension/index.bs

```diff
@@ -0,0 +1,535 @@
+      dictionary MagnetometerReadingSet {
+        required double? x;
+        required double? y;
+        required double? z;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Uncalibrated Magnetometer ### {#mock-uncalibrated-magnetometer}
+The <dfn for="MockSensorType" enum-value>"uncalibrated-magnetometer"</dfn> type is the mock sensor type
+associated with the usage of the <a>UncalibratedMagnetometer</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary UncalibratedMagnetometerReadingSet {
+        required double? x;
+        required double? y;
+        required double? z;
+        required double? xBias;
+        required double? yBias;
+        required double? zBias;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Absolute Orientation Sensor ### {#mock-absolute-orientation-sensor}
+The <dfn for="MockSensorType" enum-value>"absolute-orientation"</dfn> type is the mock sensor type
+associated with the usage of the <a>AbsoluteOrientationSensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary AbsoluteOrientationReadingSet {
+        required FrozenArray&lt;double>? quaternion;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Relative Orientation Sensor ### {#mock-relative-orientation-sensor}
+The <dfn for="MockSensorType" enum-value>"relative-orientation"</dfn> type is the mock sensor type
+associated with the usage of the <a>RelativeOrientationSensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary RelativeOrientationReadingSet : AbsoluteOrientationReadingSet {
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Geolocation Sensor ### {#mock-geolocation-sensor}
+The <dfn for="MockSensorType" enum-value>"geolocation"</dfn> type is the mock sensor type
+associated with the usage of the <a>GeolocationSensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary GeolocationReadingSet {
+        required double? latitude;
+        required double? longitude;
+        required double? altitude;
+        required double? accuracy;
+        required double? altitudeAccuracy;
+        required double? heading;
+        required double? speed;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Proximity Sensor ### {#mock-proximity-sensor}
+The <dfn for="MockSensorType" enum-value>"proximity"</dfn> type is the mock sensor type
+associated with the usage of the <a>ProximitySensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary ProximityReadingSet {
+        required double? distance;
+        required double? max;
+        required boolean? near;
+      };
+    </pre>
+  </dd>
+</dl>
+
+Extensions {#webdriver-extensions}
+=====================
+
+## Commands ## {#mock-sensor-commands}
+
+### Create mock sensor ### {#create-mock-sensor-command}
+
+<table>
```

Comment:
Can this be in the format similar to [here](https://w3c.github.io/reporting/#generate-test-report-command) with a name and prefix?